src/constants.rs

```diff
@@ -1,6 +1,9 @@
 use foundationdb::tuple::Element;
 
 pub const COUCHDB_PREFIX: &[u8; 14] = b"\xfe\x01\xfe\x00\x14\x02couchdb\x00";
+pub const CHANGE_VS_STAMP: &[u8; 3] = b"\x15\x133";
```

Comment:
I don't understand why there is two change requests? It should jsut read from one range in the db?